pcs/common/reports/codes.py

```diff
@@ -39,6 +39,9 @@
 AGENT_IMPLEMENTS_UNSUPPORTED_OCF_VERSION = M(
     "AGENT_IMPLEMENTS_UNSUPPORTED_OCF_VERSION"
```

Comment:
I believe this is not used as `UnsupportedOcfVersion` is not raised anymore

Reply:
ack, removed